polaris-react/src/components/Frame/Frame.module.css

```diff
@@ -328,14 +313,18 @@
       /* stylelint-disable-next-line polaris/conventions/polaris/custom-property-allowed-list -- private token from component */
       width: calc(
         100vw - var(--pg-navigation-width) -
-          var(--pc-app-provider-scrollbar-width) - var(--p-space-150) -
-          var(--pc-sidebar-width)
+          var(--pc-app-provider-scrollbar-width) - var(--pc-sidebar-width) -
+          var(--p-space-050)
```

Comment:
the `var(--p-space-050)` adds a gap between the scrollbar and the edge of the content so the rounded corner doesn't cut it off.

I may undo this in https://github.com/Shopify/polaris-backlog/issues/1379 when I custom style the scrollbar.

The difference in scrollbar space is also what was causing the layout shifts when turning the beta flag off and on as per [this comment](https://github.com/Shopify/polaris/pull/11945#issuecomment-2077654707).